data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	Two Heliarch agents enter the room. "Supervise the interview, we are obligated to."`
+			choice
+				`	"How long will I have to wait?"`
+					goto "how long"
+				`	"Should I stay here to accompany Anna?`
+					goto "should I accompany anna"
+			label "how long"
+			`	"Wait a few hours, you must. But quick, this will be.", says the Kimek, interviewing you.`
+			choice
+				`	"Should I stay here to accompany Anna?"`
+			label "should I accompany anna"
+			`	"To accompany her in this room, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
+			choice
+				`	(Wait in the lounge.)`
+					goto "lounge"
+				`	(Accompany Anna during the interview.)`
+			`	You choose to remain in the room. Once the Kimek asks the first question, Anna is all too happy to begin discussing the contents of her novel in great detail.`
+			choice
+				`	(Leave the room to avoid spoilers.)`
+					goto "lounge"
+				`	(Stay in the room.)`
+			`	Under the watchful eye of the two Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, the content of the book. She is still talking in fluent English, but sometimes is forced to repeat statements due to the translator having issues with deciphering her enthusiastic tone.`
+			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet (or the Kimek equivalent) and thanks you and Anna.`
+				goto "interview over"
+			label "lounge"
+			`	You leave the room and walk to the lounge. Several Kimek are chatting in their native language, while a Saryd and Arach have chosen to rest.`
+			`	After a few hours, you see Anna emerge out of the room, and walk towards you.`
+			choice
+				`	"How did the interview go?"`
+			label "interview over"
+			`	"They told me that due to the censorship and review process, I would have to wait for roughly half a year. Thankfully, they have arranged a small apartment for me to live in. You can go on your own now, but please make sure you come back. If you want to return to human space, please add a reminder to come back and meet me."`
+			choice
+				`	"Why don't you just go back to human space?"`
+					goto "why not go to human space"
+				`	"I'll make sure to do that."`
+			label "anna thanks you"
+			`	"Thanks again, <first> <last>." I'll forever be grateful to you for helping me. I never thought I'd be successful as an author, but when I met you, you helped me turn my career into something I thought I'd never see.`
+			choice
+				`	"Just helping out."`
+				`	"It's okay."`
+			`	With that, you leave the room, and eventually the building. You board the train, walk past the same series of endless shops and streets in the spaceport city, before returning to your ship. As soon as you're in your ship, you add a reminder to visit Anna within six months.`
```

Comment:
As the event timer varies, I'd not say it as specifically six months here, just make it a broader mention of some months.

Reply:
```suggestion
			`	With that, you leave the room, and eventually the building. You board the train and walk past the same series of endless shops and streets in the spaceport city before returning to your ship. As soon as you're in your ship, you add a reminder to visit Anna later.`
```